packages/smart-board/src/SmartBoard/DataToBoard.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-underscore-dangle */
+import { statistics } from '@antv/data-wizard';
+import { getDataInsights } from '@antv/lite-insight';
+import { Advisor } from '@antv/chart-advisor';
+
+import { checkInsightTypes } from '../utils';
+import { INSIGHT_TYPES, G2PLOT_TYPES } from '../const';
+
+import { SmartBoard } from './ChartToBoard';
+
+import type { Chart, InputChart, SmartBoardType, DataToBoardProps } from '../interfaces';
+
+/**
+ * DataToBoard as SmartBoard (new SB)
+ * Get Dashboard from input data
+ * @public
+ */
+export class DataToBoard {
+  private _charts: Chart[];
+
+  private _board: SmartBoard;
+
+  private _SBType: SmartBoardType;
```

Comment:
is 'CHART ' included in _SBType of DataToBoard

Reply:
> is 'CHART ' included in _SBType of DataToBoard

'chart' will be used only if user upload charts directly.

Reply:
> but there is no entrance to input charts

The corresponding interface (for type "CHART") is on the file "ChartToBoard"